core/services/standardcapabilities/delegate.go

```diff
@@ -144,26 +149,45 @@ func (d *Delegate) ServicesForSpec(ctx context.Context, spec job.Job) ([]job.Ser
 		ethKeyBundle = ethKeyBundles[0]
 	}
 
-	log.Debug("oracleFactoryConfig: ", spec.StandardCapabilitiesSpec.OracleFactory)
+	var oracleFactory core.OracleFactory
+	if d.newOracleFactoryFn != nil {
```

Comment:
@ettec Can we add a comment here to clarify that this is really only intended for use in tests

Reply:
done